### ISSUE TITLE
Latest version doesn't mach remote version

### ISSUE THREAD

**Issue by WuglyakBolgoink** [BePo65/license-report#99]
Hallo.

I found that the output returns always the same versions:

![image](https://user-images.githubusercontent.com/2271337/180621704-48e0dd71-27e6-480b-87fc-cb5b2728f633.png)

These two props return the correct values:

```js
json['dist-tags']['latest']
json['dist-tags']['next']
```

example:
![image](https://user-images.githubusercontent.com/2271337/180621762-76f2b40e-0d5b-4432-a0c3-5c49f23612b5.png)

```
packageEntry.version 7.2.12
localVersionSemverRange 7.2.12

version 7.2.12

latest version 14.1.0
next version 14.1.0-rc.0
```

but script return:
```js
{
  name: '@angular/core',
  author: 'angular',
  licenseType: 'MIT',
  link: 'git+https://github.com/angular/angular.git',
  definedVersion: '7.2.12',
  installedVersion: '7.2.12',
  remoteVersion: '7.2.12',
  comment: '7.2.12'
}
```

**Comment by BePo65** [comment]
license-report respects the range pattern of the dependency definitions in the package.json file.

In your example it looks like you don't use ranges, but only fixed definitions (`7.2.12`), displayed in the resulting table as `installed version`.  
Therefore the `installed version` must be '7.2.12', as npm repects the value in the dependency section of package.json too.  
And the `remote version` must be '`7.2.12' no matter what other versions are available on the server.

If you want to see the latest version from the server, your definition in the package.json must be something like `>=7.2.12` or even `*` (meaning: 'give me the latest version available; my code is compatible with all versions').

**Comment by WuglyakBolgoink** [comment]
Hallo @BePo65 this is no way what we should do. The versions may be or must be always static for security reasons and to be sure that all developers has same versions on all devices.

Maybe it will be better to add another property, like "Next Version"

Example from `bootstrap`:

![image](https://user-images.githubusercontent.com/2271337/180717513-2ee801ca-d80d-484a-90e4-4e49e43599d9.png)

Example from `@angulae/core`

![image](https://user-images.githubusercontent.com/2271337/180717599-03ac5761-fe18-4ebe-8a36-fafcf82991f9.png)

**Comment by BePo65** [comment]
What version information do you need? Most (all?) npm packages have a latest dist-tag (or at least it should be possible to find out, what is the highest number), but not all packages do have a next tag.

So I could agree that we should add a `latest version` field, probably as optional field, to keep backward compatibility. This would not give you the latest compatible version (as there is no range pattern, but at least the latest available version.

How about that?

**Comment by WuglyakBolgoink** [comment]
@BePo65 good idea!

**Comment by BePo65** [comment]
Solved by pr #105
